tests/DCPS/FooTest3_0/PubDriver.cpp

```diff
@@ -173,6 +174,7 @@ PubDriver::initialize(int& argc, ACE_TCHAR *argv[])
                                    ::OpenDDS::DCPS::DEFAULT_STATUS_MASK);
   TEST_CHECK (! CORBA::is_nil (publisher_.in ()));
 
+  OpenDDS::DCPS::RestoreOutputStreamState ross(std::cout);
```

Comment:
I went back and forth on that, since its just a test and almost at the end of the test it wasn't a big deal.  But I don't mind doing that.